src/App.jsx

```diff
@@ -19,7 +28,14 @@ function App() {
     <CustomRoutes noLayout>
       <Route path="/set-password" element={<SetPasswordPage/>} />
     </CustomRoutes>
-    <Resource name="profiles" list={ListGuesser} edit={EditGuesser}/>
+    <Resource name="profiles" list={ProfileList} show={ProfileShow} edit={ProfileEdit} create={ProfileCreate} icon={Group} recordRepresentation={(record) => `${record.username}`} />
+    <Resource name="drives" list={DriveList} show={DriveShow} edit={DriveEdit} create={DriveCreate} icon={DirectionsCar} />
+    <Resource name="rides" list={RideList} show={RideShow} edit={RideEdit} create={RideCreate} icon={Chair} />
+    <Resource name="profile_features" show={ProfileFeatureShow} list={ProfileFeatureList} edit={ProfileFeatureEdit} create={ProfileFeatureCreate} icon={Rule} />
+    <Resource name="reviews" list={ReviewList} show={ReviewShow} edit={ReviewEdit} create={ReviewCreate} icon={Reviews} />
+    <Resource name="chats" list={ChatList} show={ChatShow} edit={ChatEdit} create={ChatCreate} icon={Chat} />
```

Comment:
I think the "Chat" and "Message" icons look too similar, I had to double-check to see the difference. What do you think about "Forum" for the chat resource?